server/src/Middlewares/rateLimiter.ts

```diff
@@ -0,0 +1,27 @@
+import rateLimit from "express-rate-limit";
+import { Request } from "express";
+
+const rateLimitOptions = { 
```

Comment:
use the database for store rate limiting

Reply:
DONE ✅